### ISSUE TITLE
[runtime] [atmos wew] cannot execute null.return air()

### ISSUE THREAD

**Issue by karlnp** [tgstation/tgstation#22323]
```
[11:31:18]Cannot execute null.return air().
[11:31:18]proc name: reconcile air (/datum/pipeline/proc/reconcile_air)
[11:31:18]  source file: datum_pipeline.dm,213
[11:31:18]  usr: null
[11:31:18]  src: /datum/pipeline (/datum/pipeline)
[11:31:18]  call stack:
[11:31:18]/datum/pipeline (/datum/pipeline): reconcile air()
[11:31:18]/datum/pipeline (/datum/pipeline): process()
[11:31:18]Air (/datum/subsystem/air): process pipenets(0)
[11:31:18]Air (/datum/subsystem/air): fire(0)
[11:31:18]Master (/datum/controller/master): RunQueue()
[11:31:18]Master (/datum/controller/master): Loop()
[11:31:18]Master (/datum/controller/master): StartProcessing()
```
https://tgstation13.org/parsed-logs/basil/runtimes/runtime-2016-12-19.txt

possibly related: #21787 #22199 , perhaps others, i'm new

**Comment by Cyberboss** [comment]
Not related to #22199, that's just bad mapping

**Comment by karlnp** [comment]
different stack and message, though. but i guess that can easily be the case

**Comment by Cyberboss** [comment]
TL;DR pipenets are shit

**Comment by duncathan** [comment]
this runtime is the bane of my existence

**Comment by duncathan** [comment]
the solution I have in mind for these is literally to remove pipenets

**Comment by praisenarsie** [comment]
you devil

**Comment by Dennok** [comment]
Is it still relevant?

**Comment by yoyobatty** [comment]
bump.

**Comment by Dennok** [comment]
I think it fixed.

**Comment by Dennok** [comment]
Still actual?
